src/main/java/mate/academy/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotBlank
+    @Column(nullable = false)
+    private String title;
+    @NotBlank
+    @Column(nullable = false)
+    private String author;
+    @NotBlank
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @NotNull
+    @Column(nullable = false)
```

Comment:
notNull, notBlank are not `constraints`. Remove annotations that you don't know how to use. 
```suggestion
    @Column(nullable = false)
    private String title;
    @Column(nullable = false)
    private String author;
    @Column(nullable = false, unique = true)
    private String isbn;
    @Column(nullable = false)
```